src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -843,6 +864,11 @@ class device {
         ostr << "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\n";
         ostr << "Card Power\n";
         ostr << sensor_tree::get_pretty<unsigned>( "board.physical.power" ) << " W" << std::endl;
```

Comment:
Could you explain to me what is the difference b/w board.physical.power and board.physical.pcie_power? Why do I need the second power to show up in xbutil query output?